docker-compose.local.yml

```diff
@@ -1,20 +1,33 @@
 version: '3'
 services:
+  joplindb:
+    image: postgres
+    ports:
+     - "5432:5432"
+    environment:
+      POSTGRES_DB: 'joplin'
+      POSTGRES_USER: 'joplin'
+      POSTGRES_PASSWORD: '39h3zd3j9Pc57mH3QsiS6sb4'
   joplin:
     build:
       context: .
       dockerfile: Dockerfile.local
+    network_mode: "host"
     ports:
      - "8000:80"
     volumes:
-      - ".:/app"
+     - ".:/app"
     environment:
       LOAD_DATA: $LOAD_DATA
       HEROKU_KEY: $HEROKU_KEY
       DEBUG: 1
       GUNICORN_CMD_ARGS: "--reload  --reload-engine=poll"
       JANIS_URL: http://localhost:3000
       STYLEGUIDE_URL: https://cityofaustin.github.io/digital-services-style-guide
+      DEPLOYMENT_MODE: "LOCAL"
+      DATABASE_URL: "postgres://joplin:39h3zd3j9Pc57mH3QsiS6sb4@localhost:5432/joplin"
```

Comment:
Does setup on local change for us using postgres now? If so, is there any documentation or instructions we should add to the README for setup?

Reply:
This is for local use only, it should not be a security issue since it is not run on the cloud. This could probably be changed so that postgres does not require a password. I will have to play around with the settings.

Reply:
re: @mateoclarke's comment, I don't expect any of the steps in the readme to change beyond what has changed in the move to docker-compose from using a single dockerfile. If there's something more complex than that (especially regarding rebuilding/wiping the db) then we absolutely should add it to the readme

Reply:
Agreed, the steps remain the same. @briaguya @mateoclarke